src/cadmium/util/stop_words.cr

```diff
@@ -1,47 +1,20 @@
-        "co", "con", "could", "couldnt", "cry", "de", "describe", "detail", "do", "done",
-        "down", "due", "during", "each", "eg", "eight", "either", "eleven", "else", "elsewhere",
-        "empty", "enough", "etc", "even", "ever", "every", "everyone", "everything", "everywhere",
-        "except", "few", "fifteen", "fify", "fill", "find", "fire", "first", "five", "for",
-        "former", "formerly", "forty", "found", "four", "from", "front", "full", "further",
-        "get", "give", "go", "had", "has", "hasnt", "have", "he", "hence", "her", "here",
-        "hereafter", "hereby", "herein", "hereupon", "hers", "herself", "him", "himself",
-        "his", "how", "however", "hundred", "ie", "if", "in", "inc", "indeed", "interest",
-        "into", "is", "it", "its", "itself", "keep", "last", "latter", "latterly", "least",
-        "less", "ltd", "made", "many", "may", "me", "meanwhile", "might", "mill", "mine",
-        "more", "moreover", "most", "mostly", "move", "much", "must", "my", "myself", "name",
-        "namely", "neither", "never", "nevertheless", "next", "nine", "no", "nobody", "none",
-        "noone", "nor", "not", "nothing", "now", "nowhere", "of", "off", "often", "on", "once",
-        "one", "only", "onto", "or", "other", "others", "otherwise", "our", "ours", "ourselves",
-        "out", "over", "own", "part", "per", "perhaps", "please", "put", "rather", "re", "same",
-        "see", "seem", "seemed", "seeming", "seems", "serious", "several", "she", "should", "show",
-        "side", "since", "sincere", "six", "sixty", "so", "some", "somehow", "someone", "something",
-        "sometime", "sometimes", "somewhere", "still", "such", "system", "take", "ten", "than",
-        "that", "the", "their", "them", "themselves", "then", "thence", "there", "thereafter",
-        "thereby", "therefore", "therein", "thereupon", "these", "they", "thickv", "thin",
-        "third", "this", "those", "though", "three", "through", "throughout", "thru", "thus",
-        "to", "together", "too", "top", "toward", "towards", "twelve", "twenty", "two", "un",
-        "under", "until", "up", "upon", "us", "very", "via", "was", "we", "well", "were", "what",
-        "whatever", "when", "whence", "whenever", "where", "whereafter", "whereas", "whereby",
-        "wherein", "whereupon", "wherever", "whether", "which", "while", "whither", "who",
-        "whoever", "whole", "whom", "whose", "why", "will", "with", "within", "without", "would",
-        "yet", "you", "your", "yours", "yourself", "yourselves", "the",
-        "a", "b", "c", "d", "e", "f", "g", "h", "i", "j", "k", "l", "m", "n",
-        "o", "p", "q", "r", "s", "t", "u", "v", "w", "x", "y", "z", "$", "1",
-        "2", "3", "4", "5", "6", "7", "8", "9", "0", "_",
-      ]
+      @@lang : Symbol = :en
+      I18N_ALL_STOPWORDS = Hash(String, Array(String)).from_json(File.read("#{__DIR__}/../../../data/stopwords.json"))
```

Comment:
This `File.read` needs to be changed to the `file_read` macro. Small change semantically, but `File.read` will read things in at runtime, whereas `read_file` will load them in at compile time. The only problem with this approach is that no one will be able to change the path to the json file, but I'll accept this solution until we think of a better one.

Reply:
Ok, I'll try to think about your solution as I implement the `FrenchStemmer`

Reply:
Ok, done. Do you want more modifications before merging this in ?

Reply:
I'm going to do some testing with both of your PRs tomorrow, then I'll merge

Reply:
Some remarks : 

- Definitely convinced with replacing the JSON file with individual files
- However, putting the list of stop words in .cr doesn't look to me as very maintainable.
- The `Cadmium::I18n` module is a great idea.
- I'd prefer declaring a method `Cadmium::StopWords(language = en)` where `language` is the 639-1 iso code of the language or an array of language codes or the keyword "all" to load all the stop words.

I'll work on this and push some new code to this PR for you to see if that's ok :-)

Reply:
Just keep in mind it would actually have to be `Cadmium.stop_words(language: :en)` or something similar, since Crystal doesn't have the magic initializers like Ruby does. I like that idea though, it fits in with the other convenience methods that exists on the base `Cadmium` module.